### ISSUE TITLE
logging fails in Indigo 

### ISSUE THREAD

**Issue by Pandoro** [strands-project/strands_perception_people#117]
When starting the the people_tracker_robot.launch with log:=true the following error is thrown:

``` bash
process[save_people_locations-6]: started with pid [8363]
Traceback (most recent call last):
  File "/home/lucie01/bellbot_catkin_ws/src/strands_perception_people/bayes_people_tracker_logging/scripts/save_locations.py", line 11, in <module>
    from camera_calibration.approxsync import ApproximateSynchronizer
ImportError: No module named approxsync
[save_people_locations-6] process has died [pid 8363, exit code 1, cmd /home/lucie01/bellbot_catkin_ws/src/strands_perception_people/bayes_people_tracker_logging/scripts/save_locations.py __name:=save_people_locations __log:=/home/lucie01/.ros/log/dace88d8-b072-11e4-a468-00032d225877/save_people_locations-6.log].
log file: /home/lucie01/.ros/log/dace88d8-b072-11e4-a468-00032d225877/save_people_locations-6*.log
```

**Comment by cdondrup** [comment]
this should be fixed by #130, but I will test it first.

**Comment by cdondrup** [comment]
Tested. Works.
